mlflow/R/mlflow/man-roxygen/roxlate-model-uri.R

```diff
@@ -0,0 +1,11 @@
+#' @param model_uri The location, in URI format, of the MLflow model.
+#' @family artifact uri
+#' @details The URI scheme must be supported by MLflow - i.e. there has to be an MLflow artifact
+#'          repository corresponding to the scheme of the URI. The content is expected to point to a
+#'          directory containing MLmodel. The following are examples of valid model uris:
+#'                  - ``file:///Users/me/path/to/local/model``
```

Comment:
super nit: ``file:///absolute/path/to/local/model`` might make the relative one more clear.

Reply:
Good point, I'll update it.